tests/async_tests/test_interfaces.py

```diff
@@ -313,7 +316,7 @@ async def test_connection_pool_get_connection_info(
     reason="Unix Domain Sockets only exist on Unix",
 )
 @pytest.mark.usefixtures("async_environment")
-async def test_http_request_unix_domain_socket(uds_server) -> None:
+async def test_http_request_unix_domain_socket(uds_server: Server,) -> None:
```

Comment:
(Nit)

```suggestion
async def test_http_request_unix_domain_socket(uds_server: Server) -> None:
```

Reply:
Weird, I thought black would normalize this.